components/CaseStatus/CaseStatusView.tsx

```diff
@@ -21,28 +21,39 @@ const CaseStatusView = ({ person }: Props): React.ReactElement => {
 
   return (
     <>
-      {caseStatusData && (
-        <section className="govuk-!-margin-top-0">
-          {caseStatusData.caseStatuses.map((status) => (
-            <span
-              className="govuk-tag lbh-tag govuk-!-margin-right-1 govuk-!-margin-top-2"
-              key={status.id}
-            >
-              {getTypeString(status.type as keyof typeof valueMapping)} -{' '}
-              {status.fields.map((val) => val.selectedOption.name).join(',')}
-            </span>
-          ))}
-        </section>
+      {groupByType(caseStatusData.caseStatuses, (data: any) => data.type).map(
+        (status) => (
+          <span
+            className="govuk-tag lbh-tag lbh-tag--yellow govuk-!-margin-right-1 govuk-!-margin-top-2"
+            key={status}
+          >
+            {getTypeString(status as keyof typeof valueMapping)}
+          </span>
+        )
       )}
     </>
   );
 };
 
+function groupByType(list: any, keyGetter: any) {
+  const map = new Array<string>();
```

Comment:
Feels a bit confusing for the variable to be called map which is a string[] when the Map type exists